api/src/test/java/org/openmrs/module/fhir2/api/impl/FhirEncounterServiceImplTest.java

```diff
@@ -34,6 +46,28 @@
 	
 	private static final String PATIENT_IDENTIFIER = "1003GH";
 	
+	private static final String PATIENT_FULL_NAME = "Mr. John Doe";
+	
+	private static final String ENCOUNTER_INITIAL_DATA_XML = "org/openmrs/module/fhir2/api/dao/impl/FhirEncounterDaoImplTest_initial_data.xml";
```

Comment:
We shouldn't need the encounter data XML file anywhere outside of the DAO tests. Everything else should be mocked.